addons/hw_drivers/main.py

```diff
@@ -82,7 +82,7 @@ def send_alldevices(self, iot_client=None):
             except Exception:
                 _logger.exception('Could not reach configured server')
         else:
-            _logger.warning('Odoo server not set')
+            _logger.info('The IoT Box does not have an associated Odoo database')
```

Comment:
don't use "IoT box" term, it's no longer correct with windows IoT. I personally use the terme "IoT". It's also generally useless to precise (it would be like if odoo logs says "your odoo instance" evey time.
```suggestion
            _logger.info('ignore devices synchronisation with odoo database as unset')
```